README.md

```diff
@@ -1,9 +1,28 @@
-# Enact [![Travis](https://img.shields.io/travis/enactjs/enact.svg?style=flat-square)](https://travis-ci.org/enactjs/enact) [![license](https://img.shields.io/github/license/enactjs/enact.svg?style=flat-square)](http://www.apache.org/licenses/LICENSE-2.0) [![Gitter](https://img.shields.io/gitter/room/EnactJS/Lobby.svg?style=flat-square)](https://gitter.im/EnactJS/Lobby)
+<p align="center">
+  <img src="https://github.com/enactjs/enact/assets/4288375/25a32c12-6a9b-44d4-80b3-a8f82ddcc643">
+</p>
+
+<h1>An app development framework built atop React that’s easy to use, performant and customizable.</h1>
```

Comment:
🤔 How about mentioning `Enact` before this description?
```
# Enact [![Travis](https://img.shields.io/travis/com/enactjs/enact/master?style=flat-square)]
```

Reply:
Do you mean like "Enact is an app development framework...."?

Reply:
Make sense. Updated!